5-auth/news-aggregator/src/controllers/authController.js

```diff
@@ -0,0 +1,45 @@
+// controllers/authController.js
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const { ACCESS_TOKEN_SECRET } = require("../config");
+
+const users = [];
+
+const register = async (req, res) => {
+  try {
+    const salt = await bcrypt.genSalt();
+    const hashedPassword = await bcrypt.hash(req.body.password, salt);
```

Comment:
This can be taken into list of promises and can be awaited together. Right now line 10 execution completes and then line 11 execution completes. Thread is blocked on both lines 10 and 11. This can be avoided and maximum usage of async can be done by using Promise.all()
 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all)